cup_of_cake.py

```diff
@@ -0,0 +1,22 @@
+import functools
+
+
+def get_recipe_price(prices, optionals=[], **ingredients):
+    """
+
+    :param prices: dictionary of prices and products
+    :param optionals: optional ingredients to canceling
+    :param ingredients: to pick up
+    :return: total sum of products
+    """
+    total = 0
+    PRICE = 1
+    PRODUCT = 0
+    for product in ingredients.items():
```

Comment:
You can use tuple unpacking and iterate on the two values  - for key, value in ingreidients.itmes()